core/src/main/java/com/g16/feyrune/model/player/Player.java

```diff
@@ -20,6 +22,7 @@ public Player(String name, Point coordinates) {
         this.coordinates = coordinates;
         texture = new Texture("assets/hero/humanmale/humanMale.png");
         this.observers = new ArrayList<>();
+        creature = CreatureFactory.createCreature();
```

Comment:
should the Player have a connection to creature factory?

Reply:
It's a temporary fix until we implement a way for the players to acquire new creatures 🙂